source/Octopus.Tentacle.Tests.Integration/ScriptServiceV2IntegrationTest.cs

```diff
@@ -170,7 +170,7 @@ await clientTentacle.TentacleClient.ExecuteScript(startScriptCommand, scriptCanc
             stopWatch.Stop();
 
             actualException.Should().NotBeNull().And.BeOfType<OperationCanceledException>().And.Match<Exception>(x => x.Message == "Script execution was cancelled");
-            stopWatch.Elapsed.Should().BeLessOrEqualTo(TimeSpan.FromSeconds(10));
+            stopWatch.Elapsed.Should().BeLessOrEqualTo(TimeSpan.FromSeconds(15));
```

Comment:
ℹ️ This was done to reduce flakiness on Win2012 integration tests.
- It might stay/remove depending on the decision whether to run win2012 build

Reply:
Revert shi